src/main/java/org/opentripplanner/updater/vehicle_rental/datasources/GbfsFreeVehicleStatusMapper.java

```diff
@@ -61,4 +69,18 @@ public VehicleRentalVehicle mapFreeVehicleStatus(GBFSBike vehicle) {
       return null;
     }
   }
+
+  private String getName(GBFSBike vehicle) {
+    var typeId = vehicle.getVehicleTypeId();
+    if (typeId != null) {
+      var type = vehicleTypes.get(vehicle.getVehicleTypeId());
+      if (type != null && type.name != null) {
+        return type.name;
+      } else {
+        return RentalVehicleType.getDefaultType(system.systemId).name;
+      }
+    } else {
+      return "%s,%s".formatted(vehicle.getLat(), vehicle.getLon());
```

Comment:
It's not possible to do:
```suggestion
      return RentalVehicleType.getDefaultType(system.systemId).name;
```
...here as well? :thinking:

Reply:
That's a good point. I changed it in https://github.com/opentripplanner/OpenTripPlanner/pull/5013/commits/da18c6c1035312dd5320f41a95b8024c199993d7